### ISSUE TITLE
Group Support

### ISSUE THREAD

**Issue by zachbakerdev** [jmshrv/finamp#640]
Suggestion to add support for groups. Something I used a lot in Spotify and something I would love to use in Finamp.

**Comment by Chaphasilor** [comment]
I'm not sure what you mean. What exactly is a "group" in the context of a music app? 🤔

**Comment by zachbakerdev** [comment]
In Spotify, it's called a Jam. You can have two different people listening to the same music together remotely. The feature is built into Jellyfin itself, and you can test it out by opening up the web version on two devices. There's an icon at the top right with three people. Click it to create a group, then click it on the other device to join the group you created. Then you can play music and it will sync playback on both devices. Skipping songs will skip on both devices, playing new songs will start them on both devices, so on and so forth.

**Comment by Chaphasilor** [comment]
Right, that feature is called "SyncPlay" in Jellyfin. I've been interested in adding that eventually already, but mostly to enable synchronizing playback between my own devices, so that I can switch from one client to another one without losing my playback queue.

What would be your use case? I've heared the Jellyfin core team mention a few times that no-one really knows how SyncPlay works, and that it's a nightmare to maintain. So I'm not sure if I really want to implement support for it, if there isn't a major use case.

**Comment by Chaphasilor** [comment]
Okay, so your girlfriend has an account on your server, with access to the same music libraries as you? Because otherwise this would (probably) not work.  
I can definitely see that being useful, but since it's limited to accounts on the same server, I'm not sure how popular it would be.

And I don't think there's a way to "invite" someone to a server, giving them temporary read-only access?

**Comment by zachbakerdev** [comment]
Yup. That's exactly what I do. I believe you could make a temp user  using the Jellyfin API, but it wouldn't be necessary as you could just create a user and only give them access to the music library if that's what you want to do. Music is relatively low bandwidth compared to the other things, especially if you force transcoding for certain users.

**Comment by AltruisticRambler** [comment]
Hi! Hey I have this exact same scenario, but also a mix of the in-house 'casting' typa scenario. Would love to be able to share music session with my family / friends that share my libraries, and would love to add devices throughout my house to a session! Not a priority of course, but please keep this feature on the radar for someday!

**Comment by z411** [comment]
Just to add my use case, I also have the exact scenario with my girlfriend and friends - they have an account on my server and also a shared directory to add music to our library. We currently use Jellyfin-web to use Syncplay for music but Finamp is such a much better experience on Android that it'd be really great if we could use it instead.
